servers/cromwell/Dockerfile

```diff
@@ -7,15 +7,15 @@ ADD ./ /job-manager
 RUN /bin/bash -c scripts/rebuild_swagger.sh
 
 # We cannot yet adopt a blessed Python image because Job Manager uses python2 and blessed only has 3
-FROM gcr.io/google-appengine/python
+FROM python:3
```

Comment:
No need to implement immediately, but "blessed" suggests a selection from [this list](https://github.com/broadinstitute/dsp-appsec-blessed-images/tree/main/python) which are hosted [on GCR](https://console.cloud.google.com/gcr/images/broad-dsp-gcr-public/us/base/python?project=broad-dsp-gcr-public).

Reply:
Great question. In general, the strategy is to choose the most minimal one that is still sufficient for everything to work correctly.

The preference hierarchy is distroless > alpine > debian.

Reply:
So I ended up using the debian image, as the other two did not work out of the box. I think this may just be an issue because we need gcc for a python package and that is the only distro with it, so its possible we could address that directly, but I decided not to investigate further.